src/components/AmplitudeAnalytics/index.ts

```diff
@@ -0,0 +1,81 @@
+import { Identify, identify, init, track } from '@amplitude/analytics-browser'
+
+import { UserPropertyOperations } from './constants'
+
+/**
+ * Initializes Amplitude with API key for project.
+ *
+ * Uniswap has two Amplitude projects: test and production. You must be a
+ * member of the organization on Amplitude to view details.
+ */
+export function initializeAnalytics() {
+  if (process.env.NODE_ENV === 'development') return
+
+  const API_KEY = process.env.REACT_APP_AMPLITUDE_KEY
+  if (typeof API_KEY === 'undefined') {
+    throw new Error(`REACT_APP_AMPLITUDE_KEY must be a defined environment variable`)
+  }
+
+  init(
+    API_KEY,
+    undefined, // User ID should be undefined to let Amplitude default to Device ID
```

Comment:
perhaps name a variable called `const userId = undefined` to make this clearer

Reply:
Added an inline comment for clarity, for both userId and options params. Thanks for pointing out.